docs/index.rst

```diff
@@ -31,17 +31,19 @@ You can find an example project `here`_.
 
 
     class Candidate(models.Model):
-        photo = ImageField(blank=True, manual_crop="")
+        photo = ImageField(blank=True, manual_crop='4:3')
```

Comment:
I'm not sure if I'm correct here, but I think @nd0ut made something about this stuff in blocks v0.26-0.27.

Reply:
Yes, it's already been implemented. `manual_crop` will become `crop-preset` for the new uploader or `data-crop` for the old one.